### ISSUE TITLE
Qt: NVDA doesn't read correctly

### ISSUE THREAD

**Issue by OzancanKaratas** [nvaccess/nvda#7119]
With NVDA, some Qt 5 programs is not used well. For example, edit fields is not being skipped. NVDA is writing tab character instead skipping to next element.
NVDA isn't navigateing on editable combo boxes.
IAccessible APIs should be repaired.

**Comment by josephsl** [comment]
Hi, evidence (example apps, steps to reproduce) please? Thanks.

**Comment by OzancanKaratas** [comment]
This problems and more is general, but was seen in TeamTalk standard client and VLC media player.

**Comment by jcsteh** [comment]
These are issues that need to be fixed either in the application or in Qt. For example, the application and the GUI toolkit is responsible for handling the tab key, not NVDA.
